giles-eco/src/main/java/edu/asu/diging/gilesecosystem/web/core/service/processing/impl/CompletedStorageRequestProcessor.java

```diff
@@ -85,4 +87,13 @@ public String getProcessedTopic() {
     public Class<? extends ICompletedStorageRequest> getRequestClass() {
         return CompletedStorageRequest.class;
     }
+    
+    private void setOldFileVersionsIfReprocessing(IFile file) {
```

Comment:
just call it `setOldFileVersions` or something. It does not check if it's reprocessing or not, only if the storage id is null. If we reuse it in a different scenario, it would get confusing.